python/heterocl/devices.py

```diff
@@ -370,6 +389,10 @@ def fpga(cls, vendor, model):
     def gpu(cls, vendor, model):
         return GPU(vendor, model)
 
+    @classmethod
+    def ssd(cls, cap, path):
+        return SSD(cap, path)
+
```

Comment:
When will this be useful?

Reply:
When users want to create a custom platform with SSD.

Reply:
I just want to know why we need a `classmethod`. Do we have any real case scenario that is using this feature right now?

Reply:
This approach is not clean enough, just for fast prototyping. When users want to create a custom platform, they should run 
```python
        config = {
            "host" : hcl.dev.cpu("intel", "e5"),
            "xcel" : [
                hcl.dev.fpga("xilinx", "xcvu19p")
            ]
        }
```